### DISCUSSION TITLE
是否可以用Fastapi部署pp-structure架构，还是使用PaddleHub Serving来实现，想使用最新的模型

### DISCUSSION THREAD

**Discussion by LKAMING97** [PaddlePaddle/PaddleOCR#13507] [Q&A]
目前用fastapi进行服务化部署，使用多进程又报错了TypeError: can't pickle paddle.fluid.core_avx.PaddleInferPredictor objects ，不知道怎么解决，一般是不是使用他自带的PaddleHub Serving进行部署

**Comment by GreatV** [comment]
paddle serving不维护了

**Comment by LKAMING97** [comment]
那是不是自己写了要

**Reply by SWHL** [reply]
是的，可以参考我之前整理的[RapidStructure](https://github.com/RapidAI/RapidStructure)
